src/metrics/graphite_metric.ts

```diff
@@ -16,15 +16,22 @@ export class GraphiteMetric extends AbstractMetric {
     this._queryParts[1] = this._queryParts[1].split(MAX_DATA_POINTS_REGEX)[0];
   }
 
-  getQuery(from: number, to: number, limit: number, offset: number): string {
+  getQuery(from: number, to: number, limit: number, offset: number): MetricQuery {
 
     let moment_format = 'h:mm_YYYYMMDD';
     let from_date = moment(from).format(moment_format);
     let to_date = moment(to).format(moment_format);
 
 
     let timeClause = `&from=${from_date}&until=${to_date}`;
-    return `?${this._queryParts[0]}${timeClause}&${this._queryParts[1]}&maxDataPoints=${limit}`;
+    let q = `?${this._queryParts[0]}${timeClause}&${this._queryParts[1]}&maxDataPoints=${limit}`;
```

Comment:
Lets do something like: 
```
axios.get('/user', {
    params: {
      ID: 12345
    }
  })
```
to avoid building params string manually

Reply:
but it is fine if you prefer this way

Reply:
prefer this way